diagnostics/blocking_neale/doc/blocking_neale.rst

```diff
@@ -0,0 +1,250 @@
+.. This is a comment in RestructuredText format (two periods and a space).
+
+.. Note that all "statements" and "paragraphs" need to be separated by a blank 
+   line. This means the source code can be hard-wrapped to 80 columns for ease 
+   of reading. Multi-line comments or commands like this need to be indented by
+   exactly three spaces.
+
+.. Underline with '='s to set top-level heading: 
+   https://docutils.sourceforge.io/docs/user/rst/quickref.html#section-structure
+
+Rich Neale's Blocking Diagnostic Documentation
+================================
+
+Last update: 09/30/2021
+
+The diagnostic evaluates the blocking frequency in a model by longitude and season
+as determined by the meridional gradient above a threshold value of daily
+500mb height following `D'Andrea et al 1998 <https://doi.org/10.1007/s003820050230>`__.
+
+
+.. Underline with '-'s to make a second-level heading.
+
+Version & Contact info
+----------------------
+
+.. '-' starts items in a bulleted list: 
+   https://docutils.sourceforge.io/docs/user/rst/quickref.html#bullet-lists
+
+Here you should describe who contributed to the diagnostic, and who should be
+contacted for further information:
```

Comment:
```suggestion
```

Reply:
I forgot to put the link to the data on the ftp server: ftp://ftp.cgd.ucar.edu/archive/mdtf/obs_data.blocking_neale.20211001.tar